src/renderer/dx/DxRenderer.hpp

```diff
@@ -252,6 +254,7 @@ namespace Microsoft::Console::Render
         // Preferences and overrides
         bool _softwareRendering;
         bool _forceFullRepaintRendering;
+        bool _intenseIsBold{ true };

```

Comment:
nit: looks like all of the other ones are initialized in the ctor. We should probably move this there for consistency

Reply:
We should start moving these things into the header honestly.